.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 .turbo
-.vscode
```

Comment:
this is a risky move, as i have some other files here like workspace configs, and I'm pretty sure I'm not the only one.

Reply:
Good point, thanks.
```
.vscode
!.vscode/launch.json
```
seems to work like a charm on my side.